src/pages/learn/network/flow.mdx

```diff
@@ -62,9 +62,9 @@ Axelar creates and deploys a smart contract called the Gas Receiver to all conne
 
 ## Monitoring and recovery
 
-Once the message is executed on the destination chain, its cross-chain journey is complete. However, there may be factors that prevent this completion at various stages of the process. These might include a gas limit set too low, or contract logic that is incompatible with a function called on the destination chain. In order to mitigate against these obstructions, Axelar provides various tools for [debugging](https://docs.axelar.dev/dev/debug/error-debugging), [monitoring and recovery](https://docs.axelar.dev/dev/general-message-passing/monitoring). 
+Once the message is executed on the destination chain, its cross-chain journey is complete. However, there may be factors that prevent this completion at various stages of the process. These might include a gas limit set too low, or contract logic that is incompatible with a function called on the destination chain. In order to mitigate against these obstructions, Axelar provides various tools for [debugging](/dev/general-message-passing/debug/error-debugging/), [monitoring and recovery](/dev/general-message-passing/monitoring/). 
 
-For example, while a cross-chain message is being processed by Axelar, a user or developer can follow and observe its journey. The Axelarscan explorer has a General Message Passing [tracking tool](https://testnet.axelarscan.io/gmp/search) built for this purpose. The user can enter their source-chain sender address or transaction hash into the Axelarscan search bar to find their cross-chain message. This will bring up a detailed page containing information about the message, and its current status. If there are any issues with the transaction, this page will also suggest possible fixes. Visit this [link](https://docs.axelar.dev/dev/general-message-passing/monitoring) for more information on how to use this monitoring and recovery tool.
+For example, while a cross-chain message is being processed by Axelar, a user or developer can follow and observe its journey. The Axelarscan explorer has a General Message Passing [tracking tool](https://testnet.axelarscan.io/gmp/search) built for this purpose. The user can enter their source-chain sender address or transaction hash into the Axelarscan search bar to find their cross-chain message. This will bring up a detailed page containing information about the message, and its current status. If there are any issues with the transaction, this page will also suggest possible fixes. Visit this [link](/dev/general-message-passing/monitoring) for more information on how to use this monitoring and recovery tool.
```

Comment:
```suggestion
For example, while a cross-chain message is being processed by Axelar, a user or developer can follow and observe its journey. The Axelarscan explorer has a General Message Passing [tracking tool](https://testnet.axelarscan.io/gmp/search) built for this purpose. The user can enter their source-chain sender address or transaction hash into the Axelarscan search bar to find their cross-chain message. This will bring up a detailed page containing information about the message, and its current status. If there are any issues with the transaction, this page will also suggest possible fixes. Visit this [link](/dev/general-message-passing/monitoring/) for more information on how to use this monitoring and recovery tool.
```